app/tests/unit/routes/v1/user.spec.js

```diff
@@ -0,0 +1,97 @@
+const request = require('supertest');
+
+const { expressHelper } = require('../../../common/helper');
+const validator = require('../../../../src/validator/user');
+
+//
+// Mock middleware, we are not testing this here
+//
+jest.mock('../../../../src/middleware/authorization', () => ({
+  checkAppMode: jest.fn((_req, _res, next) => next()),
+}));
+
+jest.mock('../../../../src/middleware/featureToggle', () => ({
+  requireDb: jest.fn((_req, _res, next) => next()),
+  requireSomeAuth: jest.fn((_req, _res, next) => next()),
+}));
+
+jest.mock('../../../../src/validator/user');
+
+//
+// Mocks are in place, create the router
+//
+const router = require('../../../../src/routes/v1/user');
+
+const { userController } = require('../../../../src/controllers');
+
+// Express Server
+const basePath = '/api/v1/user';
+const app = expressHelper(basePath, router);
+
+// Mock config library - @see {@link https://stackoverflow.com/a/64819698}
+jest.mock('config');
+
+describe(`GET ${basePath}`, () => {
+  // TODO: Ensure other middleware funcs are called once
+  const validatorSearchUserSpy = jest.spyOn(validator, 'searchUsers');
+  const ctrlrSearchUserSpy = jest.spyOn(userController, 'searchUsers');
+
+  beforeEach(() => {
+    validatorSearchUserSpy.mockReset();
+    ctrlrSearchUserSpy.mockReset();
+  });
+
+  it('should call controller with known query params', async () => {
+    // eslint-disable-next-line no-unused-vars
+    validatorSearchUserSpy.mockImplementation((_req, _res, next) => next());
+    // eslint-disable-next-line no-unused-vars
+    ctrlrSearchUserSpy.mockImplementation((req, res, next) => res.status(200).end());
```

Comment:
Not a particular fan of having to ignore no-unused-vars repeatedly like this. If we have to continue doing this (aka no way to circumvent this due to implementation mock limits...), I'd recommend we add this following line to the very top of this file instead to reduce verbosity:
```js
/* eslint-disable no-unused-vars */
```